language/move-bytecode-verifier/kani/src/lib.rs

```diff
@@ -0,0 +1,23 @@
+use move_bytecode_verifier::verify_module;
+use move_binary_format::file_format::Bytecode;
+
+#[cfg(kani)]
+#[kani::proof]
+#[kani::unwind(1)]
+fn test_get_successors() {
```

Comment:
I guess arbitrary bytecode would not satisfy this? Do we need to run a few phases of the bytecode verifier here to basically constraint the input until the first time the CFG is used?

Reply:
When I test it, I changed get_successors back to the previous code with wrong conditions to check whether the tool can catch it but so far there is no luck (32GB memory and 16 core). I am currently trying with different parameters including unwind.

Reply:
update: if I set unwind to 100, error can be found